src/components/Header/components/LocationSearch/LocationSearch.module.scss

```diff
@@ -0,0 +1,14 @@
+.currentLocation {
+  .title {
+    color: #aaaaaa;
+  }
+
+  .value {
+    display: inline-block;
+    text-decoration: none;
+    color: #00b2ff;
+    border-bottom: 1px dashed #00b2ff;
+    padding-bottom: 1px;
+    margin-left: 5px;
```

Comment:
I suggest to never use hardcoded size values. You can use variables

Reply:
Done. I created style constants.